src/pydantic_ome_ngff/latest/multiscales.py

```diff
@@ -157,7 +157,7 @@ class MultiscaleAttrs(Attrs):
 class MultiscaleGroup(Group):
     attrs: MultiscaleAttrs
 
-    @root_validator
+    @model_validator(mode="before")
```

Comment:
Docs aren't very clear about what before/wrap/after are: I think it's whether the received `values` are as dicts or as concrete types?